<error errorId="58b83494-04b8-4499-8a4e-326f86f8f309" application="/LM/W3SVC/4/ROOT" host="SF-D001" type="System.Data.SqlClient.SqlException" message="The INSERT statement conflicted with the FOREIGN KEY constraint &quot;FK_Actions_Residents&quot;. The conflict occurred in database &quot;CHM&quot;, table &quot;dbo.Residents&quot;, column 'ID'.&#xD;&#xA;The statement has been terminated." source=".Net SqlClient Data Provider" detail="System.Web.HttpUnhandledException (0x80004005): An error occurred while updating the entries. See the inner exception for details. ---&gt; System.Data.Entity.Infrastructure.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---&gt; System.Data.Entity.Core.UpdateException: An error occurred while updating the entries. See the inner exception for details. ---&gt; System.Data.SqlClient.SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint &quot;FK_Actions_Residents&quot;. The conflict occurred in database &quot;CHM&quot;, table &quot;dbo.Residents&quot;, column 'ID'.&#xD;&#xA;The statement has been terminated.&#xD;&#xA;   at System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)&#xD;&#xA;   at System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)&#xD;&#xA;   at System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp; dataReady)&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption)&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader(Boolean isInternal, Boolean forDescribeParameterEncryption)&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.InternalEndExecuteNonQuery(IAsyncResult asyncResult, String endMethod, Boolean isInternal)&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult)&#xD;&#xA;   at System.Data.SqlClient.SqlCommand.EndExecuteNonQueryAsync(IAsyncResult asyncResult)&#xD;&#xA;   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.Core.Mapping.Update.Internal.DynamicUpdateCommand.&lt;ExecuteAsync&gt;d__0.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.&lt;UpdateAsync&gt;d__0.MoveNext()&#xD;&#xA;   --- End of inner exception stack trace ---&#xD;&#xA;   at System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.&lt;UpdateAsync&gt;d__0.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.Core.Objects.ObjectContext.&lt;ExecuteInTransactionAsync&gt;d__3d`1.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.Core.Objects.ObjectContext.&lt;SaveChangesToStoreAsync&gt;d__39.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.&lt;ExecuteAsyncImplementation&gt;d__9`1.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Data.Entity.Core.Objects.ObjectContext.&lt;SaveChangesInternalAsync&gt;d__31.MoveNext()&#xD;&#xA;   --- End of inner exception stack trace ---&#xD;&#xA;   at CHM.Services.Controllers.SyncronizationController.&lt;SaveofflineActions&gt;d__4.MoveNext() in E:\Enthusis_Latest\Enthusis.Services\CHM.Services\CHM.Services\Controllers\SyncronizationController.cs:line 138&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Threading.Tasks.TaskHelpersExtensions.&lt;CastToObject&gt;d__3`1.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Web.Http.Controllers.ApiControllerActionInvoker.&lt;InvokeActionAsyncCore&gt;d__0.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Web.Http.Controllers.ActionFilterResult.&lt;ExecuteAsync&gt;d__2.MoveNext()&#xD;&#xA;--- End of stack trace from previous location where exception was thrown ---&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)&#xD;&#xA;   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)&#xD;&#xA;   at System.Web.Http.Controllers.ExceptionFilterResult.&lt;ExecuteAsync&gt;d__0.MoveNext()" user="senak@gmail.com" time="2017-01-28T16:49:33.6983025Z" statusCode="500" webHostHtmlMessage="&lt;!DOCTYPE html&gt;&#xD;&#xA;&lt;html&gt;&#xD;&#xA;    &lt;head&gt;&#xD;&#xA;        &lt;title&gt;The INSERT statement conflicted with the FOREIGN KEY constraint &amp;quot;FK_Actions_Residents&amp;quot;. The conflict occurred in database &amp;quot;CHM&amp;quot;, table &amp;quot;dbo.Residents&amp;quot;, column 'ID'.&lt;br&gt;The statement has been terminated.&lt;/title&gt;&#xD;&#xA;        &lt;meta name=&quot;viewport&quot; content=&quot;width=device-width&quot; /&gt;&#xD;&#xA;        &lt;style&gt;&#xD;&#xA;         body {font-family:&quot;Verdana&quot;;font-weight:normal;font-size: .7em;color:black;} &#xD;&#xA;         p {font-family:&quot;Verdana&quot;;font-weight:normal;color:black;margin-top: -5px}&#xD;&#xA;         b {font-family:&quot;Verdana&quot;;font-weight:bold;color:black;margin-top: -5px}&#xD;&#xA;         H1 { font-family:&quot;Verdana&quot;;font-weight:normal;font-size:18pt;color:red }&#xD;&#xA;         H2 { font-family:&quot;Verdana&quot;;font-weight:normal;font-size:14pt;color:maroon }&#xD;&#xA;         pre {font-family:&quot;Consolas&quot;,&quot;Lucida Console&quot;,Monospace;font-size:11pt;margin:0;padding:0.5em;line-height:14pt}&#xD;&#xA;         .marker {font-weight: bold; color: black;text-decoration: none;}&#xD;&#xA;         .version {color: gray;}&#xD;&#xA;         .error {margin-bottom: 10px;}&#xD;&#xA;         .expandable { text-decoration:underline; font-weight:bold; color:navy; cursor:hand; }&#xD;&#xA;         @media screen and (max-width: 639px) {&#xD;&#xA;          pre { width: 440px; overflow: auto; white-space: pre-wrap; word-wrap: break-word; }&#xD;&#xA;         }&#xD;&#xA;         @media screen and (max-width: 479px) {&#xD;&#xA;          pre { width: 280px; }&#xD;&#xA;         }&#xD;&#xA;        &lt;/style&gt;&#xD;&#xA;    &lt;/head&gt;&#xD;&#xA;&#xD;&#xA;    &lt;body bgcolor=&quot;white&quot;&gt;&#xD;&#xA;&#xD;&#xA;            &lt;span&gt;&lt;H1&gt;Server Error in '/' Application.&lt;hr width=100% size=1 color=silver&gt;&lt;/H1&gt;&#xD;&#xA;&#xD;&#xA;            &lt;h2&gt; &lt;i&gt;The INSERT statement conflicted with the FOREIGN KEY constraint &amp;quot;FK_Actions_Residents&amp;quot;. The conflict occurred in database &amp;quot;CHM&amp;quot;, table &amp;quot;dbo.Residents&amp;quot;, column 'ID'.&lt;br&gt;The statement has been terminated.&lt;/i&gt; &lt;/h2&gt;&lt;/span&gt;&#xD;&#xA;&#xD;&#xA;            &lt;font face=&quot;Arial, Helvetica, Geneva, SunSans-Regular, sans-serif &quot;&gt;&#xD;&#xA;&#xD;&#xA;            &lt;b&gt; Description: &lt;/b&gt;An error occurred while updating the entries. See the inner exception for details.&#xD;&#xA;            &lt;br&gt;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;b&gt; Exception Details: &lt;/b&gt;System.Data.SqlClient.SqlException: The INSERT statement conflicted with the FOREIGN KEY constraint &amp;quot;FK_Actions_Residents&amp;quot;. The conflict occurred in database &amp;quot;CHM&amp;quot;, table &amp;quot;dbo.Residents&amp;quot;, column 'ID'.&lt;br&gt;The statement has been terminated.&lt;br&gt;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;b&gt;Source Error:&lt;/b&gt; &lt;br&gt;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;table width=100% bgcolor=&quot;#ffffcc&quot;&gt;&#xD;&#xA;               &lt;tr&gt;&#xD;&#xA;                  &lt;td&gt;&#xD;&#xA;                      &lt;code&gt;&lt;pre&gt;&#xD;&#xA;&#xD;&#xA;Line 136:            catch (Exception ex)&#xD;&#xA;Line 137:            {&#xD;&#xA;&lt;font color=red&gt;Line 138:                throw ex;&#xD;&#xA;&lt;/font&gt;Line 139:            }&#xD;&#xA;Line 140:&lt;/pre&gt;&lt;/code&gt;&#xD;&#xA;&#xD;&#xA;                  &lt;/td&gt;&#xD;&#xA;               &lt;/tr&gt;&#xD;&#xA;            &lt;/table&gt;&#xD;&#xA;&#xD;&#xA;            &lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;b&gt; Source File: &lt;/b&gt; E:\Enthusis_Latest\Enthusis.Services\CHM.Services\CHM.Services\Controllers\SyncronizationController.cs&lt;b&gt; &amp;nbsp;&amp;nbsp; Line: &lt;/b&gt; 138&#xD;&#xA;            &lt;br&gt;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;b&gt;Stack Trace:&lt;/b&gt; &lt;br&gt;&lt;br&gt;&#xD;&#xA;&#xD;&#xA;            &lt;table width=100% bgcolor=&quot;#ffffcc&quot;&gt;&#xD;&#xA;               &lt;tr&gt;&#xD;&#xA;                  &lt;td&gt;&#xD;&#xA;                      &lt;code&gt;&lt;pre&gt;&#xD;&#xA;&#xD;&#xA;[SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint &amp;quot;FK_Actions_Residents&amp;quot;. The conflict occurred in database &amp;quot;CHM&amp;quot;, table &amp;quot;dbo.Residents&amp;quot;, column &amp;#39;ID&amp;#39;.&#xD;&#xA;The statement has been terminated.]&#xD;&#xA;   System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction) +3277272&#xD;&#xA;   System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose) +345&#xD;&#xA;   System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean&amp;amp; dataReady) +5002&#xD;&#xA;   System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption) +492&#xD;&#xA;   System.Data.SqlClient.SqlCommand.CompleteAsyncExecuteReader(Boolean isInternal, Boolean forDescribeParameterEncryption) +340&#xD;&#xA;   System.Data.SqlClient.SqlCommand.InternalEndExecuteNonQuery(IAsyncResult asyncResult, String endMethod, Boolean isInternal) +910&#xD;&#xA;   System.Data.SqlClient.SqlCommand.EndExecuteNonQueryInternal(IAsyncResult asyncResult) +274&#xD;&#xA;   System.Data.SqlClient.SqlCommand.EndExecuteNonQueryAsync(IAsyncResult asyncResult) +181&#xD;&#xA;   System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization) +56&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.Core.Mapping.Update.Internal.&amp;lt;ExecuteAsync&amp;gt;d__0.MoveNext() +3588&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.Core.Mapping.Update.Internal.&amp;lt;UpdateAsync&amp;gt;d__0.MoveNext() +657&#xD;&#xA;&#xD;&#xA;[UpdateException: An error occurred while updating the entries. See the inner exception for details.]&#xD;&#xA;   System.Data.Entity.Core.Mapping.Update.Internal.&amp;lt;UpdateAsync&amp;gt;d__0.MoveNext() +1074&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.Core.Objects.&amp;lt;ExecuteInTransactionAsync&amp;gt;d__3d`1.MoveNext() +1164&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.Core.Objects.&amp;lt;SaveChangesToStoreAsync&amp;gt;d__39.MoveNext() +659&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.SqlServer.&amp;lt;ExecuteAsyncImplementation&amp;gt;d__9`1.MoveNext() +494&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Data.Entity.Core.Objects.&amp;lt;SaveChangesInternalAsync&amp;gt;d__31.MoveNext() +1094&#xD;&#xA;&#xD;&#xA;[DbUpdateException: An error occurred while updating the entries. See the inner exception for details.]&#xD;&#xA;   CHM.Services.Controllers.&amp;lt;SaveofflineActions&amp;gt;d__4.MoveNext() in E:\Enthusis_Latest\Enthusis.Services\CHM.Services\CHM.Services\Controllers\SyncronizationController.cs:138&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Threading.Tasks.&amp;lt;CastToObject&amp;gt;d__3`1.MoveNext() +252&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Web.Http.Controllers.&amp;lt;InvokeActionAsyncCore&amp;gt;d__0.MoveNext() +331&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Web.Http.Controllers.&amp;lt;ExecuteAsync&amp;gt;d__2.MoveNext() +611&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) +14139120&#xD;&#xA;   System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) +62&#xD;&#xA;   System.Web.Http.Controllers.&amp;lt;ExecuteAsync&amp;gt;d__0.MoveNext() +275&#xD;&#xA;&lt;/pre&gt;&lt;/code&gt;&#xD;&#xA;&#xD;&#xA;                  &lt;/td&gt;&#xD;&#xA;               &lt;/tr&gt;&#xD;&#xA;            &lt;/table&gt;&#xD;&#xA;&#xD;&#xA;            &lt;br&gt;&#xD;&#xA;&#xD;&#xA;    &lt;/body&gt;&#xD;&#xA;&lt;/html&gt;&#xD;&#xA;">
  <serverVariables>
    <item name="ALL_HTTP">
      <value string="HTTP_CONNECTION:keep-alive&#xD;&#xA;HTTP_CONTENT_LENGTH:1238&#xD;&#xA;HTTP_CONTENT_TYPE:application/json;charset=UTF-8&#xD;&#xA;HTTP_ACCEPT:application/json, text/plain, */*&#xD;&#xA;HTTP_ACCEPT_ENCODING:gzip, deflate&#xD;&#xA;HTTP_ACCEPT_LANGUAGE:en-US,en;q=0.8&#xD;&#xA;HTTP_AUTHORIZATION:Bearer d3CYDuo74XGJ4lUzCXzyZzKwxdZy14aDlvtuMNvBvqgtcKK11UvFc-YMrIBSYAs-lKx0EmWgldyPwRhzkTZQ9vICYQwiuyv5WyR48ZIx2IOLQiOYOlMsyqRgGbrX0rqG8XBVNT6lBLvj1bHMSqkbtFBEfOFPbTBRnl6_CZoiy0x-iLSjua58Pt7TVEQvki6NHn4z3uhMnGrNQ9I8q09DJt0c9JotRFhAFaCqcJ_J86utrEsgSb_dbcof1qjExYAhP9qjZb_kLfk8qafwjsyr8qgvu4TwJdQth5Hp_Xu1aW5pCDQZ6Qr5v957ttWWOXlm0v1W8sgnujnnwgXSiJ3NnRMnkTvOZawfvYHK1QiEHCdzwWSkU1GaFANDSLfwpQd_RzJ_3IemUHY0RvXhXYPmJKxyaH4emmF8_vdLNg-5pmWbiTcoPcDYx2EZxpO4G7YgcgFfnbAMfYF0nmaY8Qfr5A&#xD;&#xA;HTTP_HOST:sf-d001:1234&#xD;&#xA;HTTP_USER_AGENT:Mozilla/6.0 (iPhone; CPU iPhone OS 10_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/8.0 Mobile/10A5376e Safari/8536.25&#xD;&#xA;HTTP_ORIGIN:local://simulator&#xD;&#xA;HTTP_X_DEVTOOLS_EMULATE_NETWORK_CONDITIONS_CLIENT_ID:4EF2CA2E-6495-4BC9-AC77-B5719DE11D22&#xD;&#xA;" />
    </item>
    <item name="ALL_RAW">
      <value string="Connection: keep-alive&#xD;&#xA;Content-Length: 1238&#xD;&#xA;Content-Type: application/json;charset=UTF-8&#xD;&#xA;Accept: application/json, text/plain, */*&#xD;&#xA;Accept-Encoding: gzip, deflate&#xD;&#xA;Accept-Language: en-US,en;q=0.8&#xD;&#xA;Authorization: Bearer d3CYDuo74XGJ4lUzCXzyZzKwxdZy14aDlvtuMNvBvqgtcKK11UvFc-YMrIBSYAs-lKx0EmWgldyPwRhzkTZQ9vICYQwiuyv5WyR48ZIx2IOLQiOYOlMsyqRgGbrX0rqG8XBVNT6lBLvj1bHMSqkbtFBEfOFPbTBRnl6_CZoiy0x-iLSjua58Pt7TVEQvki6NHn4z3uhMnGrNQ9I8q09DJt0c9JotRFhAFaCqcJ_J86utrEsgSb_dbcof1qjExYAhP9qjZb_kLfk8qafwjsyr8qgvu4TwJdQth5Hp_Xu1aW5pCDQZ6Qr5v957ttWWOXlm0v1W8sgnujnnwgXSiJ3NnRMnkTvOZawfvYHK1QiEHCdzwWSkU1GaFANDSLfwpQd_RzJ_3IemUHY0RvXhXYPmJKxyaH4emmF8_vdLNg-5pmWbiTcoPcDYx2EZxpO4G7YgcgFfnbAMfYF0nmaY8Qfr5A&#xD;&#xA;Host: sf-d001:1234&#xD;&#xA;User-Agent: Mozilla/6.0 (iPhone; CPU iPhone OS 10_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/8.0 Mobile/10A5376e Safari/8536.25&#xD;&#xA;Origin: local://simulator&#xD;&#xA;X-DevTools-Emulate-Network-Conditions-Client-Id: 4EF2CA2E-6495-4BC9-AC77-B5719DE11D22&#xD;&#xA;" />
    </item>
    <item name="APPL_MD_PATH">
      <value string="/LM/W3SVC/4/ROOT" />
    </item>
    <item name="APPL_PHYSICAL_PATH">
      <value string="E:\Enthusis_Latest\Enthusis.Services\CHM.Services\CHM.Services\" />
    </item>
    <item name="AUTH_TYPE">
      <value string="Bearer" />
    </item>
    <item name="AUTH_USER">
      <value string="senak@gmail.com" />
    </item>
    <item name="AUTH_PASSWORD">
      <value string="*****" />
    </item>
    <item name="LOGON_USER">
      <value string="senak@gmail.com" />
    </item>
    <item name="REMOTE_USER">
      <value string="senak@gmail.com" />
    </item>
    <item name="CERT_COOKIE">
      <value string="" />
    </item>
    <item name="CERT_FLAGS">
      <value string="" />
    </item>
    <item name="CERT_ISSUER">
      <value string="" />
    </item>
    <item name="CERT_KEYSIZE">
      <value string="" />
    </item>
    <item name="CERT_SECRETKEYSIZE">
      <value string="" />
    </item>
    <item name="CERT_SERIALNUMBER">
      <value string="" />
    </item>
    <item name="CERT_SERVER_ISSUER">
      <value string="" />
    </item>
    <item name="CERT_SERVER_SUBJECT">
      <value string="" />
    </item>
    <item name="CERT_SUBJECT">
      <value string="" />
    </item>
    <item name="CONTENT_LENGTH">
      <value string="1238" />
    </item>
    <item name="CONTENT_TYPE">
      <value string="application/json;charset=UTF-8" />
    </item>
    <item name="GATEWAY_INTERFACE">
      <value string="CGI/1.1" />
    </item>
    <item name="HTTPS">
      <value string="off" />
    </item>
    <item name="HTTPS_KEYSIZE">
      <value string="" />
    </item>
    <item name="HTTPS_SECRETKEYSIZE">
      <value string="" />
    </item>
    <item name="HTTPS_SERVER_ISSUER">
      <value string="" />
    </item>
    <item name="HTTPS_SERVER_SUBJECT">
      <value string="" />
    </item>
    <item name="INSTANCE_ID">
      <value string="4" />
    </item>
    <item name="INSTANCE_META_PATH">
      <value string="/LM/W3SVC/4" />
    </item>
    <item name="LOCAL_ADDR">
      <value string="192.168.1.2" />
    </item>
    <item name="PATH_INFO">
      <value string="/api/Syncronization/SaveofflineActions" />
    </item>
    <item name="PATH_TRANSLATED">
      <value string="E:\Enthusis_Latest\Enthusis.Services\CHM.Services\CHM.Services\api\Syncronization\SaveofflineActions" />
    </item>
    <item name="QUERY_STRING">
      <value string="" />
    </item>
    <item name="REMOTE_ADDR">
      <value string="192.168.1.2" />
    </item>
    <item name="REMOTE_HOST">
      <value string="192.168.1.2" />
    </item>
    <item name="REMOTE_PORT">
      <value string="64854" />
    </item>
    <item name="REQUEST_METHOD">
      <value string="POST" />
    </item>
    <item name="SCRIPT_NAME">
      <value string="/api/Syncronization/SaveofflineActions" />
    </item>
    <item name="SERVER_NAME">
      <value string="sf-d001" />
    </item>
    <item name="SERVER_PORT">
      <value string="1234" />
    </item>
    <item name="SERVER_PORT_SECURE">
      <value string="0" />
    </item>
    <item name="SERVER_PROTOCOL">
      <value string="HTTP/1.1" />
    </item>
    <item name="SERVER_SOFTWARE">
      <value string="Microsoft-IIS/10.0" />
    </item>
    <item name="URL">
      <value string="/api/Syncronization/SaveofflineActions" />
    </item>
    <item name="HTTP_CONNECTION">
      <value string="keep-alive" />
    </item>
    <item name="HTTP_CONTENT_LENGTH">
      <value string="1238" />
    </item>
    <item name="HTTP_CONTENT_TYPE">
      <value string="application/json;charset=UTF-8" />
    </item>
    <item name="HTTP_ACCEPT">
      <value string="application/json, text/plain, */*" />
    </item>
    <item name="HTTP_ACCEPT_ENCODING">
      <value string="gzip, deflate" />
    </item>
    <item name="HTTP_ACCEPT_LANGUAGE">
      <value string="en-US,en;q=0.8" />
    </item>
    <item name="HTTP_AUTHORIZATION">
      <value string="Bearer d3CYDuo74XGJ4lUzCXzyZzKwxdZy14aDlvtuMNvBvqgtcKK11UvFc-YMrIBSYAs-lKx0EmWgldyPwRhzkTZQ9vICYQwiuyv5WyR48ZIx2IOLQiOYOlMsyqRgGbrX0rqG8XBVNT6lBLvj1bHMSqkbtFBEfOFPbTBRnl6_CZoiy0x-iLSjua58Pt7TVEQvki6NHn4z3uhMnGrNQ9I8q09DJt0c9JotRFhAFaCqcJ_J86utrEsgSb_dbcof1qjExYAhP9qjZb_kLfk8qafwjsyr8qgvu4TwJdQth5Hp_Xu1aW5pCDQZ6Qr5v957ttWWOXlm0v1W8sgnujnnwgXSiJ3NnRMnkTvOZawfvYHK1QiEHCdzwWSkU1GaFANDSLfwpQd_RzJ_3IemUHY0RvXhXYPmJKxyaH4emmF8_vdLNg-5pmWbiTcoPcDYx2EZxpO4G7YgcgFfnbAMfYF0nmaY8Qfr5A" />
    </item>
    <item name="HTTP_HOST">
      <value string="sf-d001:1234" />
    </item>
    <item name="HTTP_USER_AGENT">
      <value string="Mozilla/6.0 (iPhone; CPU iPhone OS 10_0 like Mac OS X) AppleWebKit/536.26 (KHTML, like Gecko) Version/8.0 Mobile/10A5376e Safari/8536.25" />
    </item>
    <item name="HTTP_ORIGIN">
      <value string="local://simulator" />
    </item>
    <item name="HTTP_X_DEVTOOLS_EMULATE_NETWORK_CONDITIONS_CLIENT_ID">
      <value string="4EF2CA2E-6495-4BC9-AC77-B5719DE11D22" />
    </item>
  </serverVariables>
</error>